### DISCUSSION TITLE
How to use wallet without GUI

### DISCUSSION THREAD

**Discussion by Shm013** [Chia-Network/chia-blockchain#2739] [General]
Hello!

I installed chia on GNU/Linux, but I don't want to use GUI at all.
I saw 'wallet' in CLI but I have some question about it.
Is it possible to use wallet without GUI?
Where exactly chia store my wallet?
How to backup my wallet?
How to transfer my wallet from one computer to another?

**Comment by firmicus** [comment]
Try "venv/bin/chia wallet -h" for list of available options ... 
RE: Backup/Transfer -> wallet resides on blockchain, not on host I guess ..

**Reply by Shm013** [reply]
I ask about analog of wallet.dat in bitcoin core. I don't understand what I need to backup to keep chia safe.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
